apps/envited-x-data-space/modules/AddAssets/AddAssets.actions.ts

```diff
@@ -31,9 +31,15 @@ export async function addAssetsForm(formData: FormData) {
     const result = assets.map(async (asset: File) => {
       const arrayBuffer = Buffer.from(await asset.arrayBuffer())
       const cid = await createFilename(arrayBuffer)
+      const checkIfAssetExists = await getAssetByCID(cid)
```

Comment:
A variable should not be a verb, only a function. A variable only statically holds something, it does not operate. 
Given you are requesting an asset the const should probably just be named `asset` and the asset(s) on line 31 could just be files (or uploads) at this point since we havent confirmed yet if they are assets.